libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -3439,13 +3472,12 @@ std::pair<float, float> WaveChannel::GetMinMax(
    if (t0 == t1)
       return results;
 
-   for (const auto &clip: GetTrack().mClips)
+   for (const auto &&clip: Intervals())
```

Comment:
Constant rvalue references are highly unusual. Why they are needed here?

Reply:
I will strike const

Reply:
Why the rvalue reference is needed here?

Reply:
Const was a mistake.

&& isn’t necessary but it is a reminder that the interval iterator dereferences to a value, not a reference as in the old version.

Interval iterators dereference to a shared pointer to a temporary structure.